.eslintrc.js

```diff
@@ -0,0 +1,205 @@
+module.exports = {
+	'root': true,
+    'env': {},
+    'extends': [
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@typescript-eslint/recommended-requiring-type-checking'
+    ],
+    'parser': '@typescript-eslint/parser',
+    'parserOptions': {
+        'project': 'tsconfig.eslint.json',
+        'sourceType': 'module'
+    },
+    'plugins': [
+        '@typescript-eslint',
+        '@typescript-eslint/tslint'
+    ],
+    'rules': {
+        '@typescript-eslint/adjacent-overload-signatures': 'error',
+        '@typescript-eslint/array-type': ['error', {'default': 'array-simple'}],
+        '@typescript-eslint/ban-types': 'error',
+        '@typescript-eslint/class-name-casing': 'error',
+        '@typescript-eslint/consistent-type-assertions': 'error',
+        '@typescript-eslint/consistent-type-definitions': 'off',
+        '@typescript-eslint/explicit-member-accessibility': [
```

Comment:
imho, this is not usefull for us, just will add more boilerplate.

Reply:
Agree, will drop.